### ISSUE TITLE
ARM firmware files missing on installed disk

### ISSUE THREAD

**Issue by kkaempf** [rancher/elemental-cli#391]
While installing Elemental works nicely on Raspberry Pi, it's missing the firmware files to boot.

When booting in live mode, this can be fixed by

```
# mount /dev/disk/by-label/COS_GRUB /mnt
# cp -a /boot/vc/* /mnt
# umount /mnt
```

This step should be integrated into `elemental install` when running on `aarch64`

**Comment by davidcassany** [comment]
@kkaempf I assume we can close this one now, right?

**Comment by kkaempf** [comment]
Yes. Fixed by rancher/elemental-cli#399
